aboutpage.html

```diff
@@ -0,0 +1,172 @@
+      </nav>
+     </header>
+
+      <!-- mobile nav list ends here -->
+
+      <!-- about section starts here -->
+
+  <main>
+    <section id="about-section">
+      <div class="container">
+        <div class="about-hero-container">
+          <div>
+            <div class="hero-headline-title">  
+              <h1 class="hero-headline-title text-accent-400 ">Read more about div <br>news right here </h1>
+            </div>
+            <div class="about-text">
+            <p>Div News is rated the number one news source for all things tech related,<br> Founded by our top contributors in 1995 we stand as the longest running news source for the tech industry.
+              We are proud of the ever growing numbers. as well as our rapidly growing contributors. Have something tech related to share? find out how you too can become a top contributor, by contributing informstion via our news lead intake system, simply send an email with the tech related topic you'd like to contribute to
+            </p>
+            </div>
+            <p>contributors intake Email
+            </p>
+            <div class="email-container">
+            <a class="email" href="mailto:name@email.com">contributor@divnews.com</a>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+
+      <!-- about section ends here -->
+
+      <!-- collaborations section starts here -->
+
+    <section id="collaborations">
+      <div class="container">
+        <div>
+          <h2 class="title fs-700 text-neutral-900 fw-bold">Lets collaborate</h2>
+          <div class="indicator"></div>
+        </div>
+        <div class="collaborations-container">
+          <div>
+            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Corrupti vitae labore expedita?<br> Aliquam consectetur assumenda laborum ullam accusantium. Aliquid voluptatum in consequatur. Delectus maxime labore,<br> numquam voluptate fuga soluta aliquam.</p>
+          </div>
+          <div >
+            <img  class="divnewsaboutlogo" src="img/divnewsaboutlogo.png" alt="divnewsaboutlogo">
+          </div>
+        </div>
+      </div>
+    </section>
+
+      <!-- collaborations section ends here -->
+    
+      <!-- past collaborators section starts here -->
+
+    <section>
+      <div class="container">
+        <div class="past-collaborators">
+          <h2 class="title fs-700 text-neutral-900 fw-bold">These are our past collaborators</h2>
+          <div class="indicator"></div>
+          <p>Join our wall of collaborators</p>
+        </div>
+        <div class="past-collaborators-container">
+          <div class="past-collab-section">
+            <img class="collabimage" src="img/pastcollabsone.png" alt="pastcollabsone">
+         
+          </div>
+          <div class="past-collab-section">
+            <img class="collabimage"src="img/pastcollabstwo.png" alt="pastcollabstwo">
+          </div>
+        </div>
+      </div>
+    </section>
```

Comment:
- [ ] Good job so far :+1:  But this design is a bit different from the original design. :cry: Please use the same `height`, `width`, and `color` `opacity` for the elements as defined in the original design. You can compare the two screenshots below :point_down: to see the difference and fix the design :smile: 

#### Original design
![Screenshot from 2022-10-28 18-52-59](https://user-images.githubusercontent.com/6615394/198712591-466808db-6620-4463-bb6c-af0841d76188.jpg)

#### Your design
![Screenshot from 2022-10-28 19-01-37](https://user-images.githubusercontent.com/6615394/198713053-dafa7a56-1b69-4208-a404-ee443559dd2a.jpg)

![Screenshot from 2022-10-28 19-01-52](https://user-images.githubusercontent.com/6615394/198713076-188a152c-0216-4e60-b6fc-810b8767f791.jpg)